### PR TITLE
ref(crons): Add placeholder and change resolution selection position

### PR DESCRIPTION
**Pull request by davidenwang**
Before:
image

After:
image

Loading Example:

https://github.com/getsentry/sentry/assets/9372512/551c8c05-ca44-4793-82f6-b95363f2b281

### CODE REVIEW

File: static/app/components/events/interfaces/crons/cronTimelineSection.tsx [hunk 1]
```diff
@@ -76,8 +77,8 @@ export function CronTimelineSection({event, organization}: Props) {
       title={t('Check-ins')}
       type="check-ins"
       help={t('A timeline of check-ins that happened before and after this event')}
+      actions={<ResolutionSelector />}
```
Comment (evanpurkhiser) [inline static/app/components/events/interfaces/crons/cronTimelineSection.tsx:80]: nice

**Review by evanpurkhiser — APPROVED** [review]

File: static/app/components/events/interfaces/crons/cronTimelineSection.tsx [hunk 2]
```diff
@@ -91,31 +92,33 @@ export function CronTimelineSection({event, organization}: Props) {
...
+                start={start}
+                end={end}
+                timeWindow={timeWindow}
+                environment={environment ?? DEFAULT_ENVIRONMENT}
+              />
+            </FadeInContainer>
+          </Fragment>
         ) : (
-          <Placeholder />
+          <FadeInContainer>
```
[the remark this answers is not in the document]
Comment (davidenwang) [reply static/app/components/events/interfaces/crons/cronTimelineSection.tsx:113]: hmm yea its on the other page, im a sucker for fade in, but i'm down to remove it from both. When the data loads quickly it is pretty tough to see the loading placeholder at all with the fade in :/

[the remark this answers is not in the document]
Comment (evanpurkhiser) [reply static/app/components/events/interfaces/crons/cronTimelineSection.tsx:113]: I think it's fine to have the fade in on the data, but we don't need it on the placeholder right?

[the remark this answers is not in the document]
Comment (davidenwang) [reply static/app/components/events/interfaces/crons/cronTimelineSection.tsx:113]: oh yea will def keep it on the data, just meant i can remove the fade in animation on the loading placeholder from both pages. will remove it here and make a PR to remove it from the listing page if that's alright @vuluongj20

**Pull request merged**